.github/actions/restore-node/action.yml

```diff
@@ -2,8 +2,8 @@ name: Agoric SDK restore Node.js build
 description: 'Set up Node.js environment and restore cached built files'
 
 inputs:
-  node-version:
-    description: 'The version of Node.js to use'
+  version-identifier:
```

Comment:
I don't see why this is better that `node-version`. It's no more an identifier than before. The term "version" does not imply a number. Mac OS Sequoia is a system version.

Consider reverting back to `node-version`.